docs/ContributionGuide.md

```diff
@@ -32,15 +33,16 @@ npm install
 While developing project you can use some predefined commands for running tests, running linter or generating coverage. 
  
 - Execute `npm run test` to run all tests and print the code coverage report.
-- Execute `npm run unit` to run [Unit tests](../test/unit)
-- Execute `npm run integration` to run [Integration tests](../test/integration) 
-- Execute `npm run e2e` to run [E2e tests](../test/e2e)
+- Execute `npm run unit` to run Unit tests.
+- Execute `npm run integration` to run Integration tests. (environment variables for [extension](../extension/docs/HowToRun.md#environment-variables) and [notification](../notification/docs/HowToRun.md#environment-variables) needs to be set)
 - Execute `npm run lint` to show lint errors in the code.
 
-> You need to set [environment variables](./DeploymentGuide.md#environment-variables) to be able to run integration and e2e tests.
- Additionally, there could be more environmental variables needed for tests. See the next sections of this documentation.
 
-#### E2E tests
+
+#### E2E tests (Extension module only)
 Following additional environment variables must be provided in order to run the E2E tests.
+
 | Name | Content | Required | Default value |
```

Comment:
Table markdown is not rendered correctly, can you fix it?

Reply:
9eaad393597f7759b75eb39d244ce62dd76dbd0e